config/init.go

```diff
@@ -0,0 +1,97 @@
+package config
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+)
+
+func Init(configPath string) error {
```

Comment:
Why is the file read and rewritten?

Reply:
path config is saved is fixed to `~/.cleisthenes/config`

user may want to use 1)default config or 2)custom config (<- user wrote)

either case (1), 2)) that configuration is saved to `~/.cleisthenes/config`

Reply:
oh i found configpath var in config/config.go . i think changing `configpath` local value name in `Init` function to other name is better.

it is same package var ( configPath in config/config.go ) and function local var (configPath in Init function). it can be confused.